web-dapp/server-lib/get_erc725_signature.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const config = require('../server-config');
+const sign = require('../server-lib/sign');
+const web3 = config.web3;
+
+const SIGNER_PRIVATE_KEY = config.signerPrivateKey;
+// Number "7" zero-padded-uint256-representation
+const CLAIM_TYPE_KYC_UINT256 = '0x0000000000000000000000000000000000000000000000000000000000000007';
+
+const getErc725Signature = (physicalAddress, destinationClaimHolderAddress) => {
+    let physicalAddressSha3 = web3.sha3(getAllTextDetailsFromPhysicalAddress(physicalAddress));
+
+    let dataToHash = Buffer.concat([
+        Buffer.from(destinationClaimHolderAddress.substr(2), 'hex'),
+        Buffer.from(CLAIM_TYPE_KYC_UINT256.substr(2), 'hex'),
+        Buffer.from(physicalAddressSha3.substr(2), 'hex'),
+    ]).toString('hex');
+
+    const { sig } = sign(dataToHash, SIGNER_PRIVATE_KEY);
+    const signature = '0x' + sig;
+
+    return {signature, physicalAddressSha3};
```

Comment:
`physicalAddressSha3` is _the sha3 hash of all the physical address text detail values (including the "name" field) joined using a `,` character_.

This is what I decided to include as the [ERC735 claim "data" field](https://github.com/ethereum/EIPs/issues/735), @phahulin what do you think about it?

Reply:
p/s: I should probably replace `725` with `735` in this file and the corresponding references. Will do that later because I don't want to loose the GitHub comment/conversation.